cron/rotateIndexes.py

```diff
@@ -81,14 +84,19 @@ def esRotateIndexes():
                         logger.debug('Creating %s index' % newindex)
                         es.create_index(newindex)
                     # set aliases: events to events-YYYYMMDD
-                    # and events-previous to events-YYYYMMDD-1 for example
+                    # and events-previous to events-YYYYMMDD-1
                     logger.debug('Setting {0} alias to index: {1}'.format(index, newindex))
-                    es.create_alias(index, newindex)
+                    alias = index
+                    index = newindex
+                    es.update_alias(oldindex, index, alias)
                     if oldindex in indices:
-                        logger.debug('Setting {0}-previous alias to index: {1}'.format(index, oldindex))
-                        es.create_alias('%s-previous' % index, oldindex)
+                        alias = 'events-previous'
```

Comment:
Does it meet expectations to hardcode in "events-previous" vs "events-YYYYMMDD-1"

Reply:
That's actually an artifact I should have removed, we plugged in the events-previous into the config file, which I forgot to PR as well. 

I will resolve this. Thanks for catching it!

Reply:
Done!